sympy/complexanalysis/Contour Integration.py

```diff
@@ -0,0 +1,39 @@
+from sympy import *
+import matplotlib.pyplot as plt
+
+class ComplexIntegral:
+
+    def __init__(self, eq, params=dict(shape="semi-circle", origin=(0,0), radius=oo, angle=pi)):
```

Comment:
For maintaining consistency in the codebase, you can use [**kwargs](https://www.digitalocean.com/community/tutorials/how-to-use-args-and-kwargs-in-python-3) for accepting parameters to function/method.

Reply:
Actually this is different code and on a different branch. How do I avoid it from being visible here?